implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheusmetrics/PrometheusConfig.java

```diff
@@ -58,6 +60,19 @@ default Duration step() {
         return getDuration(this, "step").orElse(Duration.ofMinutes(1));
     }
 
+    /**
+     * @return an instance of {@link Properties} that contains Prometheus Java Client
+     * config entries, for example
+     * {@code io.prometheus.exporter.exemplarsOnAllMetricTypes=true}.
+     * @see https://prometheus.github.io/client_java/config/config/
+     */
+    @Nullable
+    default Properties prometheusProperties() {
+        Properties properties = new Properties();
```

Comment:
I wonder if we should try parsing from `get` to populate the default Properties. See, for example, what we do in `OtlpConfig#resourceAttributes`. This would allow Micrometer users to use this with e.g. a `.properties` file and implementing only `get`.

Reply:
I think that's a good idea.

Reply:
I opened #4922 for it